### PR TITLE
Feature/103 add labs page

### PR DESCRIPTION
**Pull request by todd-dembrey**

### CODE REVIEW

File: opentech/public/funds/models.py [hunk 1]
```diff
@@ -46,12 +54,98 @@ def get_context(self, request, *args, **kwargs):
...
+    )
+    lab_type = models.ForeignKey(
+        'wagtailcore.Page',
+        blank=True,
+        null=True,
+        on_delete=models.SET_NULL,
+        related_name='+',
+    )
+    lab_link = models.URLField(blank=True, verbose_name='External link')
+    link_text = models.CharField(max_length=255, help_text='text to display on the button')
```
Comment (zerolab) [inline opentech/public/funds/models.py]: Capitalize `text`?

**Comment by zerolab** [comment]
A text nitpick. LGTM otherwise

File: opentech/public/funds/models.py (hunk 1 above)
Comment (todd-dembrey) [reply opentech/public/funds/models.py]: :scream:

**Comment by zerolab** [comment]
:shipit:

**Pull request merged**